openproblems/tasks/dimensionality_reduction/README.md

```diff
@@ -56,14 +63,11 @@ Different methods can require different pre-processing of the data. Standard
 pre-processing functions are available as part of the `tools` module. Where possible
 each **method** should first call one of these functions and use the processed `adata.X`
 slot as the input to the method. Raw counts are also stored in `adata.layers["counts"]`
-by the standard pre-processing functions, if a method performs its own pre-processing it
-should also do this for use by metrics. For most methods a standard pre-processing with
-the `log_cpm_hvg()` function is used which normalizes the expression matrix to counts
-per million (CPM), performs a log transformation and annotates highly-variable
-genes (HVGs) (as selected by scanpy's `high_variable_genes(adata, n_top_genes=1000,
-flavor="cell_ranger")`) to `adata.var["highly_variable"]`. Variants of methods can be
-created by applying different pre-processing prior to the method itself (see `phate.py`
-for an example).
+by the standard pre-processing functions, if a method performs its own pre-processing.
+For most methods a standard pre-processing from `log_cpm()`, which normalizes the
+expression matrix to counts per million (CPM), can be used directly from `adata.X`.
+Variants of methods can be created by applying different pre-processing prior to the
+method itself (see `phate.py` for an example).
```

Comment:
Do we still want to mention/encourage this? Or maybe we should add a disclaimer that it will affect metrics?

Reply:
Agree with adding a disclaimer